proto/executor.proto

```diff
@@ -145,6 +145,16 @@ message ANNQueryInfo {
 	optional double max_distance = 10 [(gogoproto.nullable) = false]; // Only for ANNQueryType==Where
 
 	optional uint32 hnsw_ef_search = 20 [(gogoproto.nullable) = false]; // Only for HNSW indexes
+
+	// If enabled, the content of TableScan's vector output column (whose ID is column_id) will not be
+	// used and TableScan must output an additional column using the Distance function (whose id
+	// is indicated by virtual_distance_col_id). Implementor does not need to really generate the
+	// Vector data column.
+	optional bool enable_distance_proj = 21 [(gogoproto.nullable) = false];
+
+	// as we remove the vector column in table_scan, we need to save the vector column type to indicate which vector type(nullable or not) 
+	// we need to read when index is not built.
+	optional ColumnInfo column = 22 [(gogoproto.nullable) = false];
```

Comment:
Please update the comment, because `column` will be always set, with carried type information, no matter or not enable_distance_proj is enabled.

Also, please move it after L138, due to the same reason as above.